lldp_dcbx_nl.c

```diff
@@ -801,17 +801,6 @@ int set_hw_app(char *ifname, appgroup_attribs *app_data)
 	return(recv_msg(DCB_CMD_SAPP, DCB_ATTR_APP, seq));
 }
 
-int run_cmd(char *cmd, ...)
-{
-	char cbuf[128];
-	va_list args;
-
-	va_start(args, cmd);
-	vsprintf(cbuf, cmd, args);
```

Comment:
Even if this function was used, it could use some improvement. Writing to a char array with vsprintf() provides no bounds checking and should definitely be replace. Good to just remove it all.